src/components/profile/settings/user-card.vue

```diff
@@ -0,0 +1,46 @@
+<template>
+    <div class="profile-user-card">
+        <div class="d-flex mb-2">
+            <div class="profile-icon rounded-circle me-1">
+                <img src="/img/demo/my-profile.png" alt="Profile Image">
+            </div>
+            <h5 class="align-self-center text-light">{{ username }}</h5>
+            <div class="my-auto mx-0 align-self-center ms-1">
+                <PhCheck :size="32" color="#28bb94" weight="bold" v-if="isUserVerified" />
```

Comment:
Add flyarrow `success` color to `_fly_arrow_variables.scss`
```scss
$success: #28bb94;
$success-rgb: 40, 187, 148;
```

use `class="text-success"` instead of `color="#28bb94"`

Reply:
Worked on this.